docs/create/kubernetes/kubernetes-operator/index-kubernetes-operator.mdx

```diff
@@ -0,0 +1,380 @@
+---
+id: index-kubernetes-operator
+title: 'Kubernetes Operator: What It Is and Why You Should Really Care About It?'
+sidebar_label: 'Kubernetes Operator: What It Is and Why You Should Really Care About It?'
+slug: /create/kubernetes/kubernetes-operator
+authors: [ajeet]
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import RedisCard from '@site/src/theme/RedisCard';
+
+![My Image](images/image1.png)
+
+[Kubernetes](https://kubernetes.io/) is popular due to its capability to deploy new apps at a faster pace. Thanks to “Infrastructure as data '' (specifically, YAML), today you can express all your Kubernetes  resources such as [Pods](https://kubernetes.io/docs/concepts/workloads/pods/), [Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), [Services](https://kubernetes.io/docs/concepts/services-networking/service/), [Volumes](https://kubernetes.io/docs/concepts/storage/volumes/), etc., in a YAML file. These default objects make it much easier for DevOps and SRE engineers to fully express their workloads without the need to learn how to write code in a programming language like Python, Java, or Ruby. 
+
+Kubernetes is designed for automation. Out of the box, you get lots of built-in automation from the core of Kubernetes. It can speed up your development process by making easy, automated deployments, updates (rolling update), and by managing your apps and services with almost zero downtime. However,  Kubernetes can’t automate the process natively for stateful application. For example, say you have a stateful workload, such as a database application, running on several nodes. If a majority of nodes go down, you’ll need to reload the database from a specific snapshot following specific steps. Using existing default objects, types, and controllers in Kubernetes, this would be impossible to achieve. 
```

Comment:
```suggestion
Kubernetes is designed for automation. Out of the box, you get lots of built-in automation from the core of Kubernetes. It can speed up your development process by making easy, automated deployments, updates (rolling update), and by managing your apps and services with almost zero downtime. However,  Kubernetes can’t automate the process natively for stateful applications. For example, say you have a stateful workload, such as a database application, running on several nodes. If a majority of nodes go down, you’ll need to reload the database from a specific snapshot following specific steps. Using existing default objects, types, and controllers in Kubernetes, this would be impossible to achieve. 
```